aviary/mission/gasp_based/ode/unsteady_solved/test/test_unsteady_alpha_thrust_iter_group.py

```diff
@@ -5,22 +5,28 @@
 from openmdao.utils.assert_utils import assert_near_equal
 
 from aviary.constants import GRAV_ENGLISH_LBM
-from aviary.interface.default_phase_info.two_dof import aero
 from aviary.mission.gasp_based.ode.params import ParamPort
 from aviary.mission.gasp_based.ode.unsteady_solved.unsteady_control_iter_group import \
     UnsteadyControlIterGroup
 from aviary.mission.gasp_based.ode.unsteady_solved.unsteady_solved_flight_conditions import \
     UnsteadySolvedFlightConditions
 from aviary.variable_info.enums import SpeedType
-from aviary.variable_info.variables import Aircraft, Dynamic
+from aviary.variable_info.options import get_option_defaults
 from variable_info.options import get_option_defaults
+from aviary.variable_info.variables import Aircraft, Dynamic
+from aviary.utils.aviary_values import AviaryValues
+from aviary.subsystems.aerodynamics.aerodynamics_builder import CoreAerodynamicsBuilder
+from aviary.variable_info.enums import LegacyCode
 
 
 class TestUnsteadyAlphaThrustIterGroup(unittest.TestCase):
 
     def _test_unsteady_alpha_thrust_iter_group(self, ground_roll=False):
         nn = 5
 
+        # just need aero subsystem
```

Comment:
I don't think this or any of the new imports are needed here.

Note: we do have to test that CoreAerodynmicsBuilder runs when called, but that would belong in a unit test, and the return should be checked.